cypress/integration/chi-js/modal.spec.js

```diff
@@ -0,0 +1,182 @@
+describe('chi-modal', function() {
+  before(() => {
+    cy.visit('http://localhost:8000/tests/js/modal.html');
+  });
+
+  describe('check base modal', () => {
```

Comment:
```suggestion
  describe('Base Modal should open and close', () => {
```
Please use more descriptive texts in the rest of the test cases as well

Reply:
Fixed in the latest commit